core/src/test/java/edu/ucr/cs/riple/core/CoreTest.java

```diff
@@ -160,6 +160,20 @@ public void multiple_field_declaration_test() {
         .start();
   }
 
+  @Test
+  public void multipleFieldDeclarationTestForceResolveEnabled() {
+    coreTestHelper
+        .addInputDirectory("test", "multiplefielddeclration")
+        .disableBailOut()
+        .addExpectedReports(
+            new TReport(
+                new OnField("Main.java", "test.Main", newHashSet("f1", "f2", "f3", "f4")), 9),
+            new TReport(new OnField("Main.java", "test.Main", singleton("f5")), 1))
+        .toDepth(1)
+        .enableForceResolve()
```

Comment:
See comment in `strictModeWithForceResolveTest()`, worth mentioning that we are not just altering the configuration, but adding extra checking here.

Reply:
Added the comment.